src/Servers/Connections.Abstractions/src/Features/IStreamListener.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Connections.Features
+{
+    public interface IStreamListener
+    {
+        ValueTask<ConnectionContext> StartUnidirectionalStreamAsync();
```

Comment:
Usually we have minimal features (like a method per interface) or something grouping related methods together. For example, a multiplexed connection would have `AcceptAsync`. The `StartBidirectionalStreamAsync` and `StartUnidirectionalStreamAsync` seem unrelated

Reply:
Yeah, I think I can break this up into 3 features: IQuicStreamListener, IStartStreamFeature (or some name), and some feature for knowing if the stream is unidirectional or bidirectional.